### PR TITLE
Integration of TTS with ElevenLabs 

### PR DESCRIPTION
**Pull request by mateusz-wosinski-ds**
ElevenLabs integration

### CODE REVIEW

File: libs/langchain/langchain/tools/eleven_labs/text2speech.py
Comment (piotr-grodek-dsai) [inline libs/langchain/langchain/tools/eleven_labs/text2speech.py]: it looks like a recursive call?

Comment (piotr-grodek-dsai) [reply libs/langchain/langchain/tools/eleven_labs/text2speech.py]: I would put import in module main once, please use also full name rather `elevenlabs.stream` is not name -clashing

File: libs/langchain/langchain/tools/eleven_labs/text2speech.py
Comment (piotr-grodek-dsai) [inline libs/langchain/langchain/tools/eleven_labs/text2speech.py:9]: I would import elevenlabs here, with error message on import.

File: libs/langchain/langchain/tools/eleven_labs/text2speech.py
Comment (piotr-grodek-dsai) [inline libs/langchain/langchain/tools/eleven_labs/text2speech.py:46]: I think it is better to decrease latency and use MemoryIO to keep it in memory, with option to dump to file later.

File: libs/langchain/langchain/tools/eleven_labs/text2speech.py
Comment (piotr-grodek-dsai) [reply libs/langchain/langchain/tools/eleven_labs/text2speech.py:9]: okay, nvm, I saw you use the module so it would crash on import there.

File: libs/langchain/langchain/tools/eleven_labs/text2speech.py
Comment (piotr-grodek-dsai) [inline libs/langchain/langchain/tools/eleven_labs/text2speech.py]: model should be a function param. 
Ideally it would be to have maybe a enumeration - but with option to pass any str if one needs to support newly released models but with nice autocompletion.

File: libs/langchain/langchain/tools/eleven_labs/text2speech.py
Comment (piotr-grodek-dsai) [inline libs/langchain/langchain/tools/eleven_labs/text2speech.py:60]: shouldn't play be an effect of tool run?

File: libs/langchain/langchain/tools/eleven_labs/text2speech.py
Comment (piotr-grodek-dsai) [inline libs/langchain/langchain/tools/eleven_labs/text2speech.py]: I am not sure what stream does. Does it play sound in your speakers?

File: libs/langchain/langchain/tools/eleven_labs/text2speech.py
Comment (piotr-grodek-dsai) [reply libs/langchain/langchain/tools/eleven_labs/text2speech.py:60]: As we talked - it conforms with another implementation.

File: libs/langchain/langchain/tools/eleven_labs/text2speech.py
Comment (piotr-grodek-dsai) [reply libs/langchain/langchain/tools/eleven_labs/text2speech.py:46]: I think to either stick for now with this implementation, to keep it consistent with another one - and later fix it in both (to decrease latency of TTS), or just do it now and promote better design - probably up to you & maintainers.

**Review by piotr-grodek-dsai — CHANGES_REQUESTED** [review]
A few points to fix and one major decision up to you what to do about in-memory IO.

**Comment by mateusz-wosinski-ds** [comment]
I would leave the implementation consistent with the other one for now, and mention our opinion in the description. 

I believe all the other comments have been addressed, please approve if it is fine

**Comment by maks-operlejn-ds** [comment]
LGTM

**Review by piotr-grodek-dsai — APPROVED** [review]
LGTM :rocket:

**Pull request merged**
